conf/server/server_full.conf

```
@@ -74,6 +74,13 @@ server {
                     # Default: false.
                     # tos_accepted = false
                 }
+
+                # Use certificate and key from disk
+                # serving_cert_file = {
+                    # cert_file_path= "conf/server/server.crt"
+                    # key_file_path = "conf/server/server.key"
```

nit: these defaults might be misleading. I'd suggest the following to make sure there is no ambiguity in the purpose of these certs:
```suggestion
                    # cert_file_path= "conf/server/bundleendpoint.crt"
                    # key_file_path = "conf/server/bundleendpoint.key"
```